5-Cookies/client-react/src/components/TodoList/TodoListItem/TodoListItem.driver.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import TodoListItem from "./TodoListItem";
+import { BaseDriver } from "../../../test/baseDriver";
+import dataHooks from "../../../dataHooks/dataHooks";
+import { mockTodo, throwErrorOnMissingWrapper } from "../../../test/utils";
+import { Item } from "../../../interfaces/interfaces";
+
+export class TodoListItemDriver extends BaseDriver {
+  private todoItem: Item = mockTodo();
+  private removeTodo: jest.Mock = jest.fn();
+
+  private buttonClickHandler = (dataHook: string) => {
+    if (!this.wrapper) return throwErrorOnMissingWrapper();
+    const buttonElem = this.wrapper.getByTestId(dataHook);
+    fireEvent.click(buttonElem);
+    return buttonElem;
+  };
+
+  private getTodoText = () => {
+    return this.wrapper?.getByText(this.todoItem.text).innerHTML;
```

Comment:
don't target dom by text, it's very fragile and can create confusion in the future (for instance what happens if the product changes and the text appears twice now?). use data hooks instead!

Reply:
I saw that in the library and thought that it was ok to use that, now I understand that it isn't, so I'll change it to data hooks